packages/website/src/components/KitDemoHero.astro

```diff
@@ -12,11 +12,10 @@ const { name } = Astro.props;
       <h1 class="heading-5 mb-6">See kits in action via our showcases</h1>
       <a
         class="btn btn-primary text-sm t-dark"
-        href={`${REPO_URL}/tree/main/starters/${name}`}
+        href="https://github.com/thisdot/starter.dev-github-showcases/"
```

Comment:
Since `REPO_URL` and `name` are no longer being used, it is probably safe to remove those imports